### DISCUSSION TITLE
Grids and points without text

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#3559] [General]
How is it possible to display grids and points (Equator, Ecliptic, Meridian, Zenith, Celestial Poles) without having the texts temporarily next to them? I ask this in order to start the tests for the students who follow the lessons in the planetarium, through direct questions during the screening. Thank you and good work
![No Labels](https://github.com/Stellarium/stellarium/assets/97767104/3cef616f-6159-46ed-b6ad-c8f03f470f3d)

**Comment by gzotti** [comment]
You cannot disable these labels.

**Comment by PLANETARAS** [comment]
and modify them? For example, by indicating only the initial (M)

**Reply by gzotti** [reply]
The strings are hardcoded. You can change the source code and compile yourself into a private version.
